docs/webpack.config.production.js

```diff
@@ -33,7 +34,9 @@ module.exports = {
       {
         test: /\.js$/,
         exclude: /(node_modules)/,
-        use: 'babel-loader',
+        use: {
+          loader: 'babel-loader?cacheDirectory=true',
```

Comment:
`loader: 'babel-loader?cacheDirectory=true'` should be used only when caching on netlify is enabled - otherwise it'll only add overhead in build time.